src/testing/grid_study.cpp

```diff
@@ -186,6 +188,62 @@ ::run_test () const
     std::vector<double> fail_conv_slop;
     std::vector<dealii::ConvergenceTable> convergence_table_vector;
 
+    // Write the convergence table to a file // add these to the parameter files as member functions /// look into creating a dictionary/map for this so were not double hard coding it
```

Comment:
Can you move this to a separate function?

Reply:
Yes I will do that, good call